hooks/pre-commit

```diff
@@ -7,4 +7,15 @@ only=`grep --color=always -rnE '(describe|it)\.only\(' ./test`
 if [ -n "${only}" ] ; then
   printf "\n\033[0;31mERROR:\033[0m Test suppression detected\n${only}\n"
   exit $( echo "${only}" | wc -l )
+fi
+
+# check there are no "console" statements
+console=`grep --color=always \
+  --exclude-dir=node_modules \
+  --exclude-dir=coverage \
+  --exclude-dir=.idea \
+  -rnE 'console\.(log|dir|debug|info|warn|error|trace)' .`
```

Comment:
Is it worth creating a `$grep` variable with all the parameters we use already in place?